src/bundle/Resources/public/js/scripts/admin.notifications.js

```diff
@@ -32,7 +32,9 @@
         notificationsContainer.append(notificationNode);
 
         if (timeout) {
-            global.setTimeout(() => notificationNode.querySelector('.ibexa-alert__close-btn').click(), timeout);
```

Comment:
Note: clicking on this button causes it to focus, which means that the focus was taken (if present on any) from another element. In case of a reported bug, the other element was the dropdown.